packages/ui/src/components/molecules/ProductCard/styles.scss

```diff
@@ -42,6 +42,11 @@
   --fs-product-card-out-of-stock-bkg-color   : var(--fs-color-disabled-bkg);
   --fs-product-card-out-of-stock-img-opacity : .5;
 
+  // Taxes label
+  --fs-product-card-taxes-label-color        : var(--fs-color-info-text);
+  --fs-product-card-taxes-text-size          : var(--fs-text-size-tiny);
+  --fs-product-card-taxes-text-weight        : var(--fs-text-weight-regular);
```

Comment:
should we add the new tokens in the docs?

Reply:
Same here